deployment/GI03FLMA_Deploy.csv

```diff
@@ -1,118 +1,118 @@
-AR30-03,,AR35-05,,GI03FLMA-RI000-00-CTDMOH100,5,1,2018-06-10T13:59:00,2019-08-11T08:25:00,CGMGI-03FLMA-00005,,CGINS-CTDMOH-12226,59.76805,-39.84252,,2600,2700,
-AR30-03,,AR35-05,,GI03FLMA-RI000-00-CTDMOH400,5,1,2018-06-10T13:59:00,2019-08-11T08:25:00,CGMGI-03FLMA-00005,,CGINS-CTDMOH-12223,59.76805,-39.84252,,2300,2700,
-AR30-03,,AR35-05,,GI03FLMA-RI000-00-CTDMOH700,5,1,2018-06-10T13:59:00,2019-08-11T08:25:00,CGMGI-03FLMA-00005,,CGINS-CTDMOH-12222,59.76805,-39.84252,,2000,2700,
-AR30-03,,AR35-05,,GI03FLMA-RI000-00-VELPTB000,5,1,2018-06-10T13:59:00,2019-08-11T08:25:00,CGMGI-03FLMA-00005,,CGINS-VELPTB-11973,59.76805,-39.84252,,1700,2700,
-AR30-03,,AR35-05,,GI03FLMA-RI000-00-VELPTB100,5,1,2018-06-10T13:59:00,2019-08-11T08:25:00,CGMGI-03FLMA-00005,,CGINS-VELPTB-12233,59.76805,-39.84252,,2600,2700,
-AR30-03,,AR35-05,,GI03FLMA-RI000-00-VELPTB400,5,1,2018-06-10T13:59:00,2019-08-11T08:25:00,CGMGI-03FLMA-00005,,CGINS-VELPTB-11976,59.76805,-39.84252,,2300,2700,
-AR30-03,,AR35-05,,GI03FLMA-RI000-00-VELPTB700,5,1,2018-06-10T13:59:00,2019-08-11T08:25:00,CGMGI-03FLMA-00005,,CGINS-VELPTB-11974,59.76805,-39.84252,,2000,2700,
-AR30-03,,AR35-05,,GI03FLMA-RIM01-00-SIOENG000,5,1,2018-06-10T13:59:00,2019-08-11T08:25:00,CGMGI-03FLMA-00005,,CGCON-SIOCON-14883,59.76805,-39.84252,,1507,2700,
-AR30-03,,AR35-05,,GI03FLMA-RIM01-02-ADCPSL003,5,1,2018-06-10T13:59:00,2019-08-11T08:25:00,CGMGI-03FLMA-00005,,CGINS-ADCPSL-20502,59.76805,-39.84252,,500,2700,
+KN221-04,,AT30-01,,GI03FLMA-RI000-00-CTDMOH000,1,1,2014-09-12T20:11:00,2015-08-18T10:41:00,CGMGI-03FLMA-00001,,CGINS-CTDMOH-12225,59.76683,-39.8425,,1700,2700,
+KN221-04,,AT30-01,,GI03FLMA-RI000-00-CTDMOH100,1,1,2014-09-12T20:11:00,2015-08-18T10:41:00,CGMGI-03FLMA-00001,,CGINS-CTDMOH-12227,59.76683,-39.8425,,2600,2700,
+KN221-04,,AT30-01,,GI03FLMA-RI000-00-CTDMOH400,1,1,2014-09-12T20:11:00,2015-08-18T10:41:00,CGMGI-03FLMA-00001,,CGINS-CTDMOH-12389,59.76683,-39.8425,,2300,2700,
+KN221-04,,AT30-01,,GI03FLMA-RI000-00-CTDMOH700,1,1,2014-09-12T20:11:00,2015-08-18T10:41:00,CGMGI-03FLMA-00001,,CGINS-CTDMOH-12222,59.76683,-39.8425,,2000,2700,
+KN221-04,,AT30-01,,GI03FLMA-RI000-00-VELPTB000,1,1,2014-09-12T20:11:00,2015-08-18T10:41:00,CGMGI-03FLMA-00001,,CGINS-VELPTB-11669,59.76683,-39.8425,,1700,2700,
+KN221-04,,AT30-01,,GI03FLMA-RI000-00-VELPTB100,1,1,2014-09-12T20:11:00,2015-08-18T10:41:00,CGMGI-03FLMA-00001,,CGINS-VELPTB-11970,59.76683,-39.8425,,2600,2700,
+KN221-04,,AT30-01,,GI03FLMA-RI000-00-VELPTB400,1,1,2014-09-12T20:11:00,2015-08-18T10:41:00,CGMGI-03FLMA-00001,,CGINS-VELPTB-11967,59.76683,-39.8425,,2300,2700,
+KN221-04,,AT30-01,,GI03FLMA-RI000-00-VELPTB700,1,1,2014-09-12T20:11:00,2015-08-18T10:41:00,CGMGI-03FLMA-00001,,CGINS-VELPTB-11964,59.76683,-39.8425,,2000,2700,
+KN221-04,,AT30-01,,GI03FLMA-RIM01-00-SIOENG000,1,1,2014-09-12T20:11:00,2015-08-18T10:41:00,CGMGI-03FLMA-00001,,CGCON-SIOCON-13286,59.76683,-39.8425,,1502,2700,

```

Comment:
Controller depth of 1504 was right.

Reply:
Hmm - I referenced Rev B of deck dwg 3602-40003 and the design depth would have the controller 6 m deeper than the CTDMO (design = 1498 m CTDMO and 1504 m Controller).  In this deployment the CTDMO was at 1496 m, so controller would be 1502 m, right?

Reply:
discussed w/ SW and will change Main Controller depth to what it originally was at 1504 m.